user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -121,50 +121,27 @@ They provide a simple and consistent way to download, configure, and install lib
 Vcpkg::
 +
 --
-To install Boost with Vcpkg, you can run a command like:
```

Comment:
The classic/manifest mode issue is independent from "People should not install all boost libraries by default.".